### DISCUSSION TITLE
editorconfig-checker linting *.vue files?

### DISCUSSION THREAD

**Discussion by jokay** [oxsecurity/megalinter#2287] [Q&A] [answered]
I would expect **editorconfig-check** not to lint `*.vue` files but we got this:

```sh
❌ Linted [EDITORCONFIG] files with [editorconfig-checker]: Found 1 error(s) - (0.67s) (expand for details)
- Using [editorconfig-checker v2.4.0] https://megalinter.io/6.18.0/descriptors/editorconfig_editorconfig_checker
- MegaLinter key: [EDITORCONFIG_EDITORCONFIG_CHECKER]
- Rules config: identified by [editorconfig-checker]
- Number of files analyzed: [292]
--Error detail:
builds/demo/website/src/src/components/molecule/DevicesContent.vue:
	42: Wrong amount of left-padding spaces(want multiple of 2)
	43: Wrong amount of left-padding spaces(want multiple of 2)
2 errors found
```

and if we make this linter happy, we get an error for **ESLint** (vue/html-indent)

![image](https://user-images.githubusercontent.com/18613935/213981900-3af7be60-8907-435c-8011-d2dc2dcc0fd1.png)

Beside it shouldn't lint `*.vue` files, I think we should follow the advice of **ESLint** (vue/html-indent) for this :wink:

**Comment by nvuillam** [comment, marked as answer]
Some linters are not always good friends :)

You may add `EDITORCONFIG_EDITORCONFIG_CHECKER_FILTER_REGEX_EXCLUDE:  (\.vue)` , so vue files will be ignored :)

**Comment by Kurt-von-Laven** [comment]
If you don't wish editorconfig-checker to lint `*.vue` files, I would make sure that you aren't instructing EditorConfig to configure your `*.vue` files via your `.editorconfig`.

**Reply by nvuillam** [reply]
Also works :)
